bugsnag-android-core/src/main/java/com/bugsnag/android/EventInternal.kt

```diff
@@ -324,4 +325,60 @@ internal class EventInternal : FeatureFlagAware, JsonStream.Streamable, Metadata
     override fun clearFeatureFlag(name: String) = featureFlags.clearFeatureFlag(name)
 
     override fun clearFeatureFlags() = featureFlags.clearFeatureFlags()
+
+    fun addError(throwError: Throwable): Error? {
+        val error = Error.createError(throwError, projectPackages, logger)
+        errors.addAll(error)
+        return error.firstOrNull()
+    }
+
+    fun addError(errorClass: String, errorMessage: String, errorType: ErrorType): Error {
```

Comment:
`errorMessage` should be nullable
```suggestion
    fun addError(errorClass: String, errorMessage: String?, errorType: ErrorType): Error {
```